frontend/tests/lighthouse.spec.ts

```diff
@@ -0,0 +1,98 @@
+import { mkdir } from "fs/promises";
+import { join } from "path";
+import { playAudit } from "playwright-lighthouse";
+import { test } from "@playwright/test";
+//import { paths } from "./axe.spec";
+import analyses from "@/fixtures/analyses.json" with { type: "json" };
+import { log } from "./util";
+
+log();
+
+export const paths = [
+  "/testbed",
+  "/", 
+  "/load-analysis", 
+  "/new-analysis",
+  "/about",
+  ...analyses.map((analysis) => `/analysis/${analysis.id}`),
+];
+
+const thresholds = {
+  performance: 80,
```

Comment:
I unfortunately have found the performance tests to be inaccurate using this library, and as such have reduced this threshold by a lot:

https://github.com/abhinaba-ghosh/playwright-lighthouse/issues/31